Areas/Identity/Pages/_ValidationScriptsPartial.cshtml

```diff
@@ -0,0 +1,18 @@
+<environment include="Development">
```

Comment:
Likewise I'm surprised this file is only now being added as part of this PR

Reply:
Please see above comment :)